project2/README.md

```diff
@@ -54,25 +54,25 @@ At this point we are going to stop and take a minute to prepare the file we will
 
 ```Python
 
-with open("movies.csv", "w", newline="") as csvfile:
-	output = csv.writer(csvfile, delimiter=",", quotechar='"', quoting=csv.QUOTE_MINIMAL)
+with open("movies.csv", "w", newline="") as CSVfile:
+	output = CSV.writer(csvfile, delimiter=",", quotechar='"', quoting=csv.QUOTE_MINIMAL)
```

Comment:
yes, this CSV won't work, unless we `import csv as CSV` which may be even more confusing for the newcomers.